code/game/objects/items.dm

```diff
@@ -1748,7 +1748,8 @@
 		return
 	if(!GLOB.embed_by_type[embed_data?.type])
 		qdel(embed_data)
-	embed_data = ispath(embed) ? get_embed_by_type(armor) : embed
+	embed_data = ispath(embed) ? get_embed_by_type(embed) : embed
+	AddElement(/datum/element/embed)
```

Comment:
Is adding the embedded element again nessassary?.

It's an embedding update so the element must have already been added somewhere for this proc to update it.

Try removing it and check if it still works

Reply:
Originally it wasnt here and it still didnt work on taped objects like kitchen knives because nothing adds it except the initialize chain. I could make it tied in to embed data being null before being changed though?

Reply:
> I could make it tied in to embed data being null before being changed though?

Yeah that sounds more ideal. We don't if all use cases call this proc just once so a safe guard like this sounds good